lib/core/rancher/step.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# IH_CORE_DIR will be set to the directory containing the bin and lib directories.
+
+function ih::setup::core.rancher::help() {
+  echo 'Install Rancher Desktop as an alternative to Docker Desktop'
+}
+
+# Check if the step has been installed and return 0 if it has.
+# Otherwise return 1.
+function ih::setup::core.rancher::test() {
+
+    if command -v rdctl; then
```

Comment:
This check is intended to determine if everything the step will accomplish has been done already. Since the step will do more than install `rdctl`, this test should also check more than that.